colossalai/shardformer/modeling/gpt2.py

```diff
@@ -331,7 +331,7 @@ def gpt2_lmhead_model_forward(
             loss_fct = CrossEntropyLoss()
             shift_logits = shift_logits.view(-1, shift_logits.size(-1))
             shift_labels = shift_labels.view(-1)
-            if shard_config.enable_tensor_parallelism:
+            if shard_config.parallel_output:
```

Comment:
`if shard_config.enable_tensor_parallelism and shard_config.parallel_output`? Default value of `shard_config.parallel_output` is True, even if tp is not enabled, right?

Reply:
updated.